### ISSUE TITLE
Indentation Issue on Helm Chart for MongoDB Replicaset v3.5.5 error validating data: [found invalid field initialDelaySeconds for v1.ExecAction]

### ISSUE THREAD

**Issue by mikeakala** [helm/charts#8088]
**Is this a request for help?**:
No
---

**Is this a BUG REPORT or FEATURE REQUEST?** (choose one):

If this is a BUG REPORT, please:
  -  I believe this is a bug related to indentation on the helm chart template for `mongodb-statefulset.yaml` template
```
          livenessProbe:
            exec:
              command:
                - sh
                - -ec
                - >-
                {{- if .Values.auth.enabled }}
                  export MONGODB_URL=mongodb://$METRICS_USER:$METRICS_PASSWORD@localhost:{{ .Values.port }};
                {{- else }}
                  export MONGODB_URL=mongodb://localhost:{{ .Values.port }};
                {{- end }}
                  /bin/mongodb_exporter
                {{- if .Values.tls.enabled }}
                  -mongodb.tls
                  -mongodb.tls-ca=/ca/tls.crt
                  -mongodb.tls-cert=/work-dir/mongo.pem
                {{- end }}
                  -test
              initialDelaySeconds: 30
              periodSeconds: 10

```
essentially an error around line 274
kindly review

**Version of Helm and Kubernetes**:

**Which chart**:

**What happened**:

**What you expected to happen**:

**How to reproduce it** (as minimally and precisely as possible):

**Anything else we need to know**:

**Comment by steven-sheehy** [comment]
Believe this is already fixed via PR #7287. Please try upgrading to latest chart (3.5.7) and close if it resolves it for you.

**Comment by mikeakala** [comment]
ok thanks for the update
